music.js

```diff
@@ -1,17 +1,28 @@
 // Define the Artist constructor function here, with a 'name' property that can be set at instantiation,
 // and a 'songs' property that is an empty array to start
-function Artist() {
-
+function Artist(name) {
+  this.name = name;
+  this.song = [];
```

Comment:
For convention purposes, this should be `this.songs`. Plural since it will be hold an array of songs.